src/pages/login/Login.tsx

```diff
@@ -0,0 +1,40 @@
+import Button from "../../components/button/Button";
+import "./Login.css";
+import background from "../../assets/background-spheres.png";
+import Footer from "../../components/footer/Footer";
+import SpheresBackground from "./spheres-background/SpheresBackground";
+import { useNavigate } from "react-router-dom";
+
+const Login: React.FC = () => {
+	const navigate = useNavigate();
+	return (
+		<div className="container">
```

Comment:
Add class name for page, so that all css under this page can be targeted specifically. See also comment in Login.css
```suggestion
		<div className="login container">
```